### PR TITLE
fix issue related to printQuality in printInBackground

### PR DESCRIPTION
**Pull request by mucahiddayan**

### CODE REVIEW

File: src/helpers/print.js [hunk 1]
```diff
@@ -94,11 +94,11 @@ const printPdf = () =>
       });
   });
 
-export const creatingPages = (pagesToPrint, includeComments, includeAnnot, printQualty, sortStrategy, clrMap, dateFormat, onProgress) => {
+export const creatingPages = (pagesToPrint, includeComments, includeAnnot, printQuality, sortStrategy, clrMap, dateFormat, onProgress) => {
   const createdPages = [];
   pendingCanvases = [];
   includeAnnotations = includeAnnot;
-  printQuality = printQualty;
+  PRINT_QUALITY = printQuality;
```
Comment (mbaer3000) [inline src/helpers/print.js:103]: Would it suffice to drop this line and simply have `multiplier: printQuality` in line 189?

File: src/helpers/print.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const print = async(dispatch, isEmbedPrintSupported, sortStrategy, colorM
       pagesToPrint,
       includeComments,
       includeAnnotations,
-      printQuality,
+      printQuality||PRINT_QUALITY,
```
Comment (mbaer3000) [inline src/helpers/print.js:54]: Better have default assignment in line 20, as in `({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal, printQuality = PRINT_QUALITY} = options)`

File: src/helpers/print.js [hunk 3]
```diff
@@ -10,14 +10,14 @@ import core from 'core';
...
+let PRINT_QUALITY = 1;
 let colorMap;
 
 export const print = async(dispatch, isEmbedPrintSupported, sortStrategy, colorMap, options) => {
-  let includeAnnotations, includeComments, pagesToPrint, onProgress;
+  let includeAnnotations, includeComments, pagesToPrint, onProgress, printQuality;
   let printWithoutModal = false;
   if (options) {
-    ({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal } = options);
+    ({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal, printQuality } = options);
```
Comment (mbaer3000) [inline src/helpers/print.js:21]: Drop if condition and rather have default assignment in method signature, as in `async(dispatch, isEmbedPrintSupported, sortStrategy, colorMap, options = {}) => {…}`

File: src/helpers/print.js [hunk 4]
```diff
@@ -10,14 +10,14 @@ import core from 'core';
 
 let pendingCanvases = [];
 let includeAnnotations = false;
-let printQuality = 1;
+let PRINT_QUALITY = 1;
```
Comment (mbaer3000) [inline src/helpers/print.js:13]: We should name this `DEFAULT_ PRINT_QUALITY`

**Comment by mucahiddayan** [comment]
newer PR exists

**Pull request closed without merging**